includes/tutor-general-functions.php

```diff
@@ -790,20 +790,23 @@ function tutor_js_date_format_against_wp() {
 	}
 }
 
-/**
- * Convert date to desire format
- *
- * NOTE: mysql query use formated date from here
- * that's why date_i18n need to be ignore
- *
- * @param $format string
- *
- * @param $date string
- *
- * @return string ( date )
-*/
 if ( ! function_exists( 'tutor_get_formated_date' ) ) {
-	function tutor_get_formated_date( string $require_format, string $user_date ) {
+	/**
+	 * Convert date to desire format
+	 *
+	 * NOTE: mysql query use formated date from here
+	 * that's why date_i18n need to be ignore
+	 *
+	 * @param string $require_format string If empty Y-m-d is used.
+	 * @param string $user_date string Date.
+	 *
+	 * @return string ( date )
+	 */
+	function tutor_get_formated_date( string $require_format = '', string $user_date ) {
+		if ( '' === $require_format ) {
+			$require_format = 'Y-m-d';
+		}
```

Comment:
We could use ternary shorthand.
```suggestion
		$require_format = $require_format ?: 'Y-m-d';
```